kolibri/core/tasks/api.py

```diff
@@ -579,14 +581,18 @@ def startexportlogcsv(self, request):
         :returns: An object with the job information
 
         """
-        csv_export_filenames = {
-            "session": "content_session_logs.csv",
-            "summary": "content_summary_logs.csv",
-        }
+        facility_id = request.data.get("facility", None)
+        if facility_id:
+            facility = Facility.objects.get(pk=facility_id)
+        else:
+            facility = request.user.facility
+
         log_type = request.data.get("logtype", "summary")
-        if log_type in csv_export_filenames.keys():
+        if log_type in CSV_EXPORT_FILENAMES.keys():
             logs_dir = os.path.join(conf.KOLIBRI_HOME, "log_export")
-            filepath = os.path.join(logs_dir, csv_export_filenames[log_type])
+            filepath = os.path.join(
+                logs_dir, CSV_EXPORT_FILENAMES[log_type].format(facility.name)
```

Comment:
Like on the UI, could you add the first four characters of the facility ID to help disambiguate facilities that have the same name. e.g. `home facility_abc12_session_logs.csv`

Reply:
Done